multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/model/SchemaFieldObject.java

```diff
@@ -6,11 +6,13 @@
 
 package com.sngular.api.generator.plugin.asyncapi.model;
 
+import java.math.BigDecimal;
 import java.util.List;
 
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.Builder.Default;
```

Comment:
Please, check the `CheckStyle` rules, I think this import should be in the previous line

Reply:
Done